lib/client/api_login_test.go

```diff
@@ -341,7 +341,8 @@ type standaloneBundle struct {
 }
 
 // TODO(codingllama): Consider refactoring newStandaloneTeleport into a public
-//  function and reusing in other places.
+//
```

Comment:
Is that extra empty line needed?

Reply:
@jakule, that's all `go fmt ./...`. I don't think it's worth chasing the formatter around.